.evergreen/config.yml

```diff
@@ -555,12 +555,16 @@ functions:
       params:
         working_dir: "src"
         script: |
-          # Disable xtrace (just in case it was accidentally set).
+          # Disable xtrace for security reasons (just in case it was accidentally set).
           set +x
+
           . ./prepare_atlas_connectivity.sh
           rm -f ./prepare_atlas_connectivity.sh
 
-          PYTHON_BINARY=${PYTHON_BINARY} bash ${PROJECT_DIRECTORY}/.evergreen/run-atlas-tests.sh
+          set -o errexit
+          set -o xtrace
+          export LC_ALL=en_US.UTF-8
+          ${PYTHON_BINARY} -m tox -e test-atlas
```

Comment:
Should this use the new run-tox script now?

Reply:
I was planning to do that in a tiny future PR to separate out the two separate tickets involved since there's a lot of changes between these changes and the tox.sh script.